js/main.js

```diff
@@ -29,12 +29,10 @@ function apiSearch(customerInput) {
   var originalUrl = 'https://tasty.p.rapidapi.com/recipes/list?from=0&size=8&q=';
   originalUrl += customerInput;
 
-  xhr.addEventListener('readystatechange', function () {
-    if (this.readyState === this.DONE) {
-      var jsonResponse = JSON.parse(this.responseText);
-      var results = jsonResponse.results;
-      parseResponse(results);
-    }
+  xhr.addEventListener('load', function () {
+    var jsonResponse = JSON.parse(this.responseText);
```

Comment:
If you assign a value of `"json"` to the `responseType` property on `xhr`, this likely won't be necessary